src/Core/src/Platform/Android/LocalizedDigitsKeyListener.cs

```diff
@@ -215,7 +215,7 @@ bool IsDecimalPointChar(char c)
 				}
 			}
 
-			return stripped ?? filterFormatted ?? new String("");
+			return stripped ?? filterFormatted;
```

Comment:
This appears to fix the issue, but can you help me understand why? Why wasn't it working before with other system display languages and why does it work now with this change?

Reply:
Yeah, Why??? 😭